docs/potato_paper.md

```diff
@@ -0,0 +1,111 @@
+# Potato Paper
+
+## Proof-of-Potato
+
+Proof-of-Potato (PoP) is a protocol for running decentralized sharded off-chain state machines.
+PoP is managed by a potato smart contract hosted on a secure blockchain such as Ethereum.
+The potato smart contract manages potato shard ownership and list of potato verifiers.
+Each potato shard runs a blockchain based potato state machine using our novel Proof-of-Tater (PoT) potato consensus algorithm.
+Other potato consensus protocols are possible too.
+We choose PoT because it offers lightweight security, speed, simplicity, and energy-efficient consensus...
+Just like a potato.
+
+### Potato Contract
+
+The potato contract is a potato smart contract.
+ The blockchain hosting the potato contract is called the potato main chain.
+
+
+**Potato Verifiers**:
+ Users aspiring to be potato verifiers can stake in potatoes in the PC to become farmers.
+ The contract manages a sorted list of all stakes over all time and has a public verifiable function assigning each potato shard to 5 potato verifiers.
```

Comment:
Okay, this also assumes that verifers are online 24/7. I was thinking of maintaining a more active list where people are more free to connect/disconnect. That makes it harder of course :P

Reply:
Oh like if verifiers were on a less expensive chain than ethereum then they could join/leave as they connect/disconnect?

Reply:
random idea I just had, but I don't think we really need 3/5 verifiers signing. As long as verifiers rotate often, you can kind of vote on whether a past transaction is good or not based on whether you choose to build off of it or not. Assuming majority verifiers are honest, I think you get the same results so long as you are willing to wait long enough for enough verifiers to have built a block on the shard. This could work better with the whole not enough verifiers being online issue.

Reply:
Ya, i was thinking for a game people would just verify while they are playing, and then maybe leave the client running for a hour or two after.